### PR TITLE
Fixes for issues related to TV devices and profiles

### PR DESCRIPTION
**Pull request by icecrystal23**
Requires spaceship pull request to be accepted first: https://github.com/fastlane/spaceship/pull/138
Fixes issue https://github.com/KrauseFx/sigh/issues/163

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 pkg
 Gemfile.lock
 fastlane/report.xml
+.project
```
Comment (KrauseFx) [inline .gitignore]: Why did you add this? Not part of the pull request, right?

File: lib/sigh/runner.rb [hunk 2]
```diff
@@ -28,7 +28,7 @@ def run
             Helper.log.info "Updating the provisioning profile".yellow
           else
             Helper.log.info "Updating the profile to include all devices".yellow
-            profile.devices = Spaceship.device.all
+            profile.devices = Spaceship.device.all_for_profile_type(profile.type)
```
Comment (KrauseFx) [inline lib/sigh/runner.rb]: Great idea :+1:

**Comment by KrauseFx** [comment]
Thanks again, merging this and removing the changes in the `.gitignore`

**Comment by icecrystal23** [comment]
Okay. Those were just files that showed up as I was developing/testing so I figured everyone would want to ignore them. I can ignore them locally anyway. Thanks!

**Comment by KrauseFx** [comment]
Thanks for the great pull request - next time please don't do a version bump :+1:

**Comment by icecrystal23** [comment]
Ok. Some projects want them. I'll try to remember you don't ;)
